adr/style/style.css

```
@@ -0,0 +1,107 @@
+
+:root{
+  --paper:#E6E8E4; --panel:#F1F2EF; --ink:#14171A; --soft:#4A5158;
+  --rule:#C3C7C1; --rule-soft:#D8DBD6;
+  --blue:#1F4E79; --amber:#7A5B12; --red:#8C2F2F; --violet:#52356B;
+  --mono: ui-monospace,"SF Mono","JetBrains Mono",Menlo,Consolas,monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)